src/media/js/views/search.js

```diff
@@ -128,6 +128,9 @@ define('views/search',
             return;
         }
         $q.trigger('blur');
+        if(!query) {
+            query = gettext('Everything');
```

Comment:
you can't change the actual query to be "Everything" because then it will [search the site for apps containing the word "Everything"](https://marketplace.firefox.com/search?q=Everything):

![image](https://cloud.githubusercontent.com/assets/203725/10537078/67fa6566-73a4-11e5-81d3-c77b0f6471db.png)

you'll likely want to handle the string change in the [template](https://github.com/mozilla/fireplace/blob/13c0b612/src/templates/search.html#L8).

<hr>

also, I saw [this comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1135210#c6), but should probably confirm that it should actually say this with "Everything" in quotation marks:

<img width="842" alt="screenshot 2015-10-16 01 26 19" src="https://cloud.githubusercontent.com/assets/203725/10537184/ebd254d4-73a4-11e5-9e44-d8f3434c197e.png">